src/CommunityVoices/Model/Entity/Group.php

```diff
@@ -57,16 +57,15 @@ public function setType($type)
         }
     }
 
-    public function validateForUpload(FlexibleObserver $stateObserver)
+    public function validateForUpload(FlexibleObserver $stateObserver, bool $childIsValid)
```

Comment:
Can you give this a default value?  I'd imagine we don't provide a value everywhere in the code.